metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/rest/client/EnrichmentClient.java

```diff
@@ -63,4 +60,8 @@ public EnrichmentBase getByUri(String uri) {
 
 		return response.getBody();
     }
+    
+    public void setRestTemplate(RestTemplate template) {
```

Comment:
Make this method default (instead of public)? Only the test code needs to have access.

Reply:
Fixed.